Apps/phairlockdigital/airlockdigital.json

```diff
@@ -0,0 +1,2075 @@
+                    "column_name": "Status",
+                    "column_order": 3,
+                    "example_values": [
+                        "success",
+                        "failed"
+                    ]
+                },
+                {
+                    "data_path": "action_result.summary",
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "action_result.message",
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "summary.total_objects",
+                    "data_type": "numeric",
+                    "example_values": [
+                        20
+                    ]
+                },
+                {
+                    "data_path": "summary.total_objects_successful",
+                    "data_type": "numeric",
+                    "example_values": [
+                        10
+                    ]
+                }
+            ],
+            "render": {
+                "type": "table"
+            },
+            "versions": "EQ(*)"
+        },
+        {
+            "action": "list identifiers",
+            "description": "Gathers a list of identifiers which are required for referencing groups, applications, baselines, or blocklists in other actions",
+            "verbose": "This action gathers the identifiers which reference the groups, applications, baselines, or blocklists within Airlock. These identifiers are required to call other actions within the Phantom in order to interact with the Airlock Server.",
+            "type": "investigate",
+            "identifier": "list_identifiers",
+            "read_only": true,
+            "parameters": {
+                "policy_type": {
+                    "description": "The type can be group, application, baseline, or blocklist",
+                    "data_type": "string",
+                    "required": true,
+                    "primary": true,
+                    "contains": [
+                        "airlockdigital policytype"
+                    ],
+                    "value_list": [
+                        "application",
+                        "baseline",
+                        "blocklist",
+                        "group"
+                    ],
+                    "order": 0
+                }
+            },
+            "output": [
+                {
+                    "data_path": "action_result.parameter.policy_type",
+                    "contains": [
+                        "airlockdigital policytype"
+                    ],
+                    "data_type": "string",
+                    "column_name": "Policy Type",
+                    "column_order": 4,
+                    "example_values": [
+                        "group"
+                    ]
+                },
+                {
+                    "data_path": "action_result.data.*.*.id",
```

Comment:
Can we iterate through the nested data in the connector file. Better to avoid `*.*.` implementation.

Reply:
It's better but for this, we have to modify the response parsing. And, if we change, then it will affect the already created playbooks of the customers. We suggest that we should avoid this change, and yes, this is a community app, so we will not able to test this change(if we add). Please suggest.

Reply:
I am more than happy to provide a login to an Airlock instance and product for validation. Or if you would like to make the change to be best practice, I can validate that the changes are not breaking. As this is the first submission of the app, there aren't really any existing playbooks to break.

Reply:
As per the above discussion we have iterated through the nested data in 'list identifiers' and 'list endpoints' to avoid *.* implementation in output datapath for these actions. @cottinghamd Could you please check the positive scenario for these actions and let us know if you find any discrepancy regarding this.

Reply:
@gusdadiya-crest I have tested 'list identifiers' and 'list endpoints' with the latest commits and they are returning data correctly and functional. I have also re-validated all the endpoints as of the latest commit and can confirm they are functional.

Reply:
@cottinghamd It's good, Thanks for confirming.